rpc_jobs/rpc_gating_project_builds.yml

```diff
@@ -0,0 +1,29 @@
+- job:
+    name: 'RPC-Gating-Project-Builds'
+    project-type: workflow
+    concurrent: true
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_all.*|.*recheck_project_builds.*'
```

Comment:
_cit_ was only added in rpc-openstack jobs to distinguish between old and new jenkins. rpc-gating has only ever had new jenkins jobs, so there's no need for a differentiator. Now that jenkins.propter is fully retired, _cit_ could be removed from the rpc-openstack jobs as well (or maybe make it optional).